MaxMind.MinFraud/Request/CreditCard.cs

```diff
@@ -107,19 +128,37 @@ public sealed class CreditCard
         }
 
         /// <summary>
-        /// The last four digits of the credit card number.
+        /// The last two or four digits of the credit card number.
         /// </summary>
-        [JsonPropertyName("last_4_digits")]
+        [Obsolete("Last4Digits is obsolete. Use LastDigits instead.")]
+        [JsonIgnore]
         public string? Last4Digits
         {
-            get => _last4Digits;
+            get => _lastDigits;
+            init
+            {
+                if (value != null && !LastDigitsRe.IsMatch(value))
+                {
+                    throw new ArgumentException($"The last credit card digits {value} is of the wrong format.");
+                }
+                _lastDigits = value;
```

Comment:
Any reason not to just do the following?

```suggestion
                Last4Digits = value;
```

It might make sense to have the `get` be `get => LastDigits;` too in case the behavior of that ever changes.